mage_ai/api/presenters/BlockPresenter.py

```diff
@@ -58,15 +56,17 @@ async def present(self, **kwargs):
                 state_stream=state_stream,
             )
 
-            if 'dbt' == display_format:
-                upstream_blocks = add_blocks_upstream_from_refs(
-                    self.model,
-                    add_current_block=True,
-                    read_only=True,
-                )
-                query_string = compiled_query_string(self.model)
+            if isinstance(self.model, DBTBlock):
```

Comment:
we only need to return the lineage and sql when the `_format` is `dbt` in the request.

Reply:
done